osmaxx/excerptexport/models/output_file.py

```diff
@@ -36,11 +47,24 @@ def file_extension(self):
     def has_file(self):
         return bool(self.file)
 
-    def __str__(self):
-        return \
-            '[' + str(self.id) + '] ' \
-            + ('file: ' + os.path.basename(self.file.name) + ', ' if (self.file and self.file.name) else '') \
-            + 'identifier: ' + str(self.public_identifier)
+    def _remove_file(self, save=False):
+        if self.file:
+            file_path = self.file.path
+            file_directory = os.path.dirname(file_path)
+            if os.path.exists(file_directory):
+                shutil.rmtree(file_directory)
```

Comment:
Maybe assert that our file is the only file in the directory, so that this won't bite us hard if we ever change the directory structure.